Reference/Management/Services/UserService/Create-a-new-user.md

```diff
@@ -0,0 +1,85 @@
+# Create a new user
+
+The document illustrates how you can create a new backoffice user from a Razor view. 
+
+
+### Getting a reference to the UserService
+The first step is to get a reference to the `UserService`. This is useful, as we're using in multiple times, and therefore doesn't have to reference it through `ApplicationContext.Current.Services.UserService` each time:
+
+    // Get a reference to the user service
+    IUserService us = ApplicationContext.Current.Services.UserService;
```

Comment:
I don't agree with telling people to use Singletons. In fact in 95% of cases there's no reason to use singletons at all. All of these services are exposed on base classes which in most cases people will be using. Even the ServiceContext is exposed as a `Services`  property.  We should be encouraging people to use all of the services we expose on base classes instead since they are all readily available there including: ApplicationContext, ServiceContext, UmbracoHelper, UmbracoContext, etc... and the base classes range from Views, controllers, web services, http handlers, etc...

Reply:
Good point. The examples should be updated to use either `Services.UserService` or  `ApplicationContext.Services.UserService` ;)